Automattic/newspack-theme
# feat: merge RAS-ACC work into trunk

### All Submissions:


### Changes proposed in this Pull Request:

This PR merges the theme changes for RAS-ACC into trunk.

For the theme, [the changes were CSS](https://github.com/Automattic/newspack-theme/compare/feat/merge-ras-acc?expand=1#diff-7752d830bcb82355072ae3b3cebb5e5bd4bce92a641a3256b5909861805f473f) to make sure the theme's font sizing didn't interfere with the Newspack UI font sizing. These are contained changes (the variables from Newspack UI are redefined in the theme and adjusted for the base 20px font size), so they won't affect anything outside of RAS-ACC.

### How to test the changes in this Pull Request:

1. Apply this and other PRs related to RAS-ACC (there should be a [plugin](https://github.com/Automattic/newspack-plugin/pull/3582), [newsletters](https://github.com/Automattic/newspack-newsletters/pull/1716), and [blocks](https://github.com/Automattic/newspack-blocks/pull/1977) one)
2. When logged in, append ?ui-demo to any page -- this should add demo content to the bottom of the page. 
3. Spot-check the font sizes -- particularly the headings -- against [the pixel sizes listed here](https://github.com/Automattic/newspack-theme/compare/feat/merge-ras-acc?expand=1#diff-7752d830bcb82355072ae3b3cebb5e5bd4bce92a641a3256b5909861805f473f) to make sure they match.

### Other information:
